### PR TITLE
[batch] make ubuntu:20.04, 22.04, 24.04 public images

### PR DESCRIPTION
**Pull request by danking**
And mirror them too!

### CODE REVIEW

File: hail/python/hailtop/batch/publicly_available_images.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    'ubuntu:24.04',
+]
+DOCKER_HUB_IMAGES_MIRRORED_IN_HAIL_BATCH = HAIL_GENETICS_IMAGES + NON_HAILGENETICS_MIRRORED_IMAGES
+
+
+def publicly_available_images(docker_prefix: str) -> List[str]:
+    return [docker_prefix + '/' + image_name for image_name in DOCKER_HUB_IMAGES_MIRRORED_IN_HAIL_BATCH]
+
+
+def is_docker_hub_image_mirrored_in_hail_batch(image_ref: ParsedDockerImageReference) -> bool:
```
Comment (daniel-goldstein) [inline hail/python/hailtop/batch/publicly_available_images.py:19]: Shouldn't this be *not* mirrored?

**Review by daniel-goldstein — CHANGES_REQUESTED** [review]
Looks great, but I think there's a missing negative

**Comment by daniel-goldstein** [comment]
Also can you dev deploy `mirror_hailgenetics_images` (maybe change that name too) to make sure that still works?

**Comment by danking** [comment]
In the spirit of not writing code in February, I'm gonna leave this as future work for the team.

**Pull request closed without merging**